src/components/buttons/gradient-border-buttons/GradientButton.jsx

```diff
@@ -32,30 +51,35 @@ function GradientBorderButtonBase({
           `hover:text-hubblr-turquoise hover:bg-black`
         }
       >
-        <div className={`${textDivClassName} flex justify-center text-lg`}>
-          <span>{children}</span>
+        <div className={`${textDivClassName} flex justify-center text-lg px-8 py-1 `}>
+          <span>
+            {children}
+            {image}
+          </span>
         </div>
       </div>
     </button>
   );
 }
 
-GradientBorderButtonBase.propTypes = {
+GradientButton.propTypes = {
   children: PropTypes.node.isRequired,
   isSubmitButton: PropTypes.bool,
-  gradientColors: PropTypes.arrayOf(PropTypes.string).isRequired,
+  image: PropTypes.element,
```

Comment:
Evtl könnte man hier das "image" auch rausnehmen aus den Props und Bilder mit in den children übergeben

Reply:
Das ist eine gute Idee - der Button mit dem Arrow Image (einziger mit Image) hat eh einen Bug. Dann könnte man schauen, ob das nicht sogar den Bug fixt!